<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="editorLayout" stretch="0,1,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="editorTitle">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Add Repeating Off Period</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="userFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="employeeLabel">
         <property name="text">
          <string>Employee</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="monthLabel">
         <property name="text">
          <string>Month</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="yearLabel">
         <property name="text">
          <string>Year</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="employeeBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="weekdayLabel">
         <property name="text">
          <string>Weekday</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="monthBox"/>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="yearBox"/>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="weekdayBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
